x-pack/test/functional_enterprise_search/README.md

```diff
@@ -0,0 +1,41 @@
+# Enterprise Search Functional E2E Tests
+
+## Running these tests
+
+Follow the [Functional Test Runner instructions](https://www.elastic.co/guide/en/kibana/current/development-functional-tests.html#_running_functional_tests).
+
+There are two suites available to run, a suite that requires a Kibana instance without an `enterpriseSearch.host`
+configured, and one that does. The later also [requires a running Enterprise Search instance](#enterprise-search-requirement), and a Private API key
+from that instance set in an Environment variable.
+
+Ex.
+
+```
+# Run specs from the x-pack directory
+cd x-pack
+
+# Run tests that require enterpriseSearch.host variable
+APP_SEARCH_API_KEY=[use private key from local App Search instance here] node scripts/functional_tests --config test/functional_enterprise_search/with_host_configured.config.ts
+
+# Run tests that do not require enterpriseSearch.host variable
+node scripts/functional_tests --config test/functional_enterprise_search/without_host_configured.config.ts
```

Comment:
I'm running this and getting a `Please provide a valid APP_SEARCH_API_KEY. See README for more details.` error in my terminal - any advice?

Reply:
I ran this with the `APP_SEARCH_API_KEY=[xxx]` prefix as in the above example and the test passed after that.

No strong feelings either way, but I'd suggest either removing the req for `APP_SEARCH_API_KEY` for the without_host config, or updating this command to include it.

Reply:
BTW - can I ask that we add `bash` right after the ``` code blocks in this markdown file? That way all the code snippets get syntax highlighting :)

Reply:
Yes, good call